identity-service/src/routes/trackListens.js

```diff
@@ -436,4 +437,52 @@ module.exports = function (app) {
       listenMap
     })
   }))
+
+  /**
+   * Gets user listen records in bulk.
+   *
+   * GET query parameters:
+   *  startTime (string) the start time to fetch listens from (in unix seconds format)
+   *  limit (number) the number of records to fetch from each user listens and anonymous listens
+   */
+  app.get('/listens/bulk', handleResponse(async (req, res) => {
+    let { startTime, limit } = req.query
+    if (!startTime || !limit) {
```

Comment:
enforcing limit seems unnecessary since you set it to min(5000, override limit) later. my suggestion would be if they pass in a limit, make sure it's < 5000 or error if > 5000.

Reply:
min (5000, limit) = 5000 if limit > 5000
but i'm cool w/ throwing an error too. i'll just do that

Reply:
oh nvm sry i read wrong. i see what you're saying. ya, still will just return an error

Reply:
my point was just to alert the user that they exceeded the bounds of the limit and if they should expect to get or should be notified. but i'm fine with whatever since this probably won't be used externally

Reply:
yayaya good point